packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -18,25 +23,84 @@ const sfdxCoreExports = vscode.extensions.getExtension(
   'salesforce.salesforcedx-vscode-core'
 )!.exports;
 const {
+  channelService,
+  notificationService,
   telemetryService,
   SfdxCommandlet,
   EmptyParametersGatherer,
-  SfdxWorkspaceChecker
+  SfdxWorkspaceChecker,
+  sfdxCoreSettings
 } = sfdxCoreExports;
 
+enum PreviewPlatformType {
+  Desktop = 1,
+  Android,
+  iOS
+}
+
+interface PreviewQuickPickItem extends vscode.QuickPickItem {
+  label: string;
+  detail: string;
+  alwaysShow: boolean;
+  picked: boolean;
+  id: PreviewPlatformType;
+  defaultTargetName: string;
+  platformName: string;
+}
+
+export const platformOptions: PreviewQuickPickItem[] = [
+  {
+    label: nls.localize('force_lightning_lwc_preview_desktop_label'),
+    detail: nls.localize('force_lightning_lwc_preview_desktop_description'),
+    alwaysShow: true,
+    picked: true,
+    id: PreviewPlatformType.Desktop,
+    platformName: '',
+    defaultTargetName: ''
+  },
+  {
+    label: nls.localize('force_lightning_lwc_android_label'),
+    detail: nls.localize('force_lightning_lwc_android_description'),
+    alwaysShow: true,
+    picked: false,
+    id: PreviewPlatformType.Android,
+    platformName: 'Android',
```

Comment:
We might want to define a type or [enum](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) for this, preferably [const enum](https://www.typescriptlang.org/docs/handbook/enums.html#const-enums).